src/graphql/data/Token.ts

```diff
@@ -48,23 +48,24 @@ export const tokenQuery = graphql`
         twitterName
         logoUrl
         tokens {
```

Comment:
one diff with relay is apollo won't add `id` automatically, which makes it impossible to properly merge cache entries with same id